src/lexers/julia.jl

```diff
@@ -146,14 +146,14 @@ julia_is_triple_string_macro(ctx::Context) = julia_is_string_macro(ctx, 3)
             ],
             :strings => [
                 (r"\"", STRING, :__pop__),
-                (r"\\([\\\"\'\$nrbtfav]|(x|u|U)[a-fA-F0-9]+|\d+)", STRING_ESCAPE),
```

Comment:
I found a bug in Weave examples where this is needed: `"\\\\"` rendered incorrectly. I couldn't find a reason why this breaks in 0.7 in the release notes though.